src/Enums.js

```diff
@@ -6,6 +6,23 @@ export const membershipStatus = [
   { key: 'Admin', value: 2 }
 ]
 
+export const officerRole = [
```

Comment:
To truly make this an enum, don't use an array. [This example](https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/) helps

Reply:
then I can't map out the children

Reply:
My intention is that it allows me to map-out each child and can get each child's value